src/dashboards/components/dashboard_index/DashboardCards.tsx

```diff
@@ -120,6 +130,9 @@ class DashboardCards extends PureComponent<OwnProps & StateProps> {
                 updatedAt={meta.updatedAt}
                 description={description}
                 onFilterChange={onFilterChange}
+                isPinned={
+                  !!pinnedItems.find(item => item?.metadata.dashboardID === id)
+                }
```

Comment:
I assume we're not concerned about every dashboard cell running this `.find` because there will only ever be 5 pinned items right?

Reply:
although it would be dope regardless to convert this to an object so that we can change this from O(n) to O(1) lookup